From eded3fd498b820c033d865bb2a61640da803d8ef Mon Sep 17 00:00:00 2001
From: Tiejun Chen <tiejun.chen@windriver.com>
Date: Fri, 28 Aug 2009 16:36:07 +0800
Subject: [PATCH 03/38] powerpc/83xx: Add the default clock frequency for mpc837x

Set the correct clock frequency for the PIT clock source for MPC837x.

Signed-off-by: Tiejun Chen <tiejun.chen@windriver.com>
---
 arch/powerpc/platforms/83xx/pit_hwtimer.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/platforms/83xx/pit_hwtimer.c b/arch/powerpc/platforms/83xx/pit_hwtimer.c
index a7fe763..206f37d 100644
--- a/arch/powerpc/platforms/83xx/pit_hwtimer.c
+++ b/arch/powerpc/platforms/83xx/pit_hwtimer.c
@@ -32,6 +32,8 @@
 #define	PIT_INTERNAL_CLOCK	(66666666 * 4)
 #elif defined(CONFIG_MPC831x_RDB)
 #define	PIT_INTERNAL_CLOCK	166666660
+#elif defined(CONFIG_PPC_MPC837x)
+#define	PIT_INTERNAL_CLOCK	333333333
 #endif
 #define	PIT_EXTERNAL_CLOCK	(32768)
 #define	PIT_DEFAULT_CLOCK	PIT_INTERNAL_CLOCK
-- 
1.6.3.3

